### DISCUSSION TITLE
[MINT]Green amulet

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1485] [General]
@adamhannigan "A failed transaction/error indicates that the item sold out before you could mint it. There were roughly 1,000 players going for the remaining 350 amulets. Unfortunately this means we could not all get items this item"

Unless you pay? So you get early access.

"Based on feedback from the last drop, we are experimenting with a new style this week! We want to reward loyal players and give them the chance to get an 'early mint'."

Based on this real players you should rethink about mint / events. This is a temporary event but you still want to strengthen a select group and dev team/artists. This game is not for a community,  it's for the devs and "friends".

If today the BIG make the game unsustainable, imagine in the future.

Lets PAY to PLAY....

**Comment by drumdrumdrum3337** [comment]
@future2back, just my 2c: it's all about balance, isn't it? 

I guess it is very important to support people who bring in significant investments into the game (and some of those investments don't even go directly into the game but are in the form of trading between players on OS), as without outside money no game would be able to survive, that's just how it is.

At the same time, I agree that those who invest more shouldn't gain everything. 

So, if there are let's say 750 items, and 50 go to folks who invested a lot, and then the rest 700 are distributed on FCFS basis among the rest of the players, then it just sounds like the right balance, doesn't it? You don't have to pay, you can take your chances and try to mint with everyone else, and if you're lucky then you are in the same boat with folks who invested much more. So, not sure you are completely fair here mate.

**Comment by adamhannigan** [comment]
@future2back Thanks for the Discussion.

Do you want to add this into the Mint/Airdrop discussion that is currently going on? This will help other people find the comment
